docs/tutorials/diafiltration-solution.ipynb

```diff
@@ -662,22 +566,14 @@
     "\n",
     "For this example, we will make use of the ``MSContactorInitializer``, an ``Initializer`` written specifically to initializer multi-stream contactor models. This initializer uses a simple sequential process to initialize the models states based on the inlet conditions for each stream and should be suitable for this example.\n",
     "\n",
-    "To sue the initializer, we first create an instance of it, and then call the ``initializer.initialize`` method providing the model we wish to initialize as an argument. Note that ``Initializers`` are generally specific to a type of model rather than a specific instance, and thus we will be able to use the same instance of this ``Initializer`` later to initializer the other membrane stages."
+    "To use the initializer, we first create an instance of it, and then call the ``initializer.initialize`` method providing the model we wish to initialize as an argument. Note that ``Initializers`` are generally specific to a type of model rather than a specific instance, and thus we will be able to use the same instance of this ``Initializer`` later to initializer the other membrane stages. The following code will generate warnings as a result of volumetric flow being set to 0, which is outside of the specified bounds, but these warnings can be ignored."
```

Comment:
Note that these warnings can be resolved by using the BT Initializer in place of the MSContactor Initializer